starter/.gitignore

```diff
@@ -0,0 +1,2 @@
+*target
+node_modules
```

Comment:
The `.gitignore` won't ignore "node_modules" since it isn't in the "starter" directory (i.e., `node_modules` is under the "ui" directory). See the description in the following resource: https://git-scm.com/docs/gitignore.

Specify where to ignore the `node_modules` since they still show up in the pull request.